app/src/main/java/com/restaurantroulette/dto/Restaurant.kt

```diff
@@ -0,0 +1,10 @@
+package com.restaurantroulette.dto
+
+data class Restaurant(var id : Int, var name : String, var address : String, var state : String,
+                 var city : String, var area : String, var postal_code : String, var country : String, var phone : String,
```

Comment:
There are a looooooot of attributes here. Some can be combined into another class (like address, which includes city state zip), and then added here.  Others should be provided with default values.  Otherwise, you can only construct this object by passing a value for each of these parameters.